webpack.js

```diff
@@ -11,6 +11,21 @@ const config = {
 	output: {
 		filename: 'terms_of_service_[name].js',
 	},
+	module: {
+		rules: [
+			{
+				test: /\.(png|jpg|gif|svg)$/,
+				loader: 'file-loader',
+				options: {
+					name: '[name].[ext]?[hash]',
+				},
+			},
+		],
+	},
+	resolve: {
+		extensions: ['*', '.js', '.vue'],
+		symlinks: false,
+	},
```

Comment:
Is this actually correct with the webpack config merge thingy @skjnldsv ?
It generates js/close.svg instead of including it, am I missing something?

Reply:
I think Julius included it in latest webpack global config release :)